### PR TITLE
docs: cli --help completion for kumactl

### LINKED ISSUE #362: include usage examples into `--help` messages of all `kumactl` commands
### Summary

* include usage examples into `--help` messages of all `kumactl` commands

### Steps To Reproduce

1. Run
```
kumactl config control-planes add --help
```
the output will be
```
Add a Control Plane.

Usage:
  kumactl config control-planes add [flags]

Flags:
      --address string   URL of the Control Plane API Server (required)
  -h, --help             help for add
      --name string      reference name for the Control Plane (required)

...
```

### Expected behaviour

* `--help` message should include `Examples:` section, e.g.

```
kumactl install database-schema --help
```

```
Install Kuma schema on DB.

Usage:
  kumactl install database-schema [flags]

Examples:

1. kumactl install database-schema --target=postgres | PGPASSWORD=mysecretpassword psql -h localhost -U postgres

2. sql_file=$(mktemp) ; \ 
kumactl install database-schema --target=postgres >$sql_file ; \
psql --host=localhost --username=postgres --password --file=$sql_file ; \
rm $sql_file

Flags:
  -h, --help            help for database-schema
      --target string   Database type: one of postgres (default "postgres")
...
```

### PR DESCRIPTION
**Pull request by Gabitchov**
### Summary

Add help documentation for remaining command without one.

### Full changelog
- [x] install control-plane
- [x] inspect dataplane
- [x] get traffic-permissions
- [x] get traffic-logs
- [x] get meshes
- [x] get proxy-templates
- [x] get trafficroutes
- [x] get dataplanes
- [x] generate dataplane-tokens
- [x] delete
- [x] config view
- [x] config control-planes switch
- [x] config control-planes add
- [x] config control-planes remove
- [ ] config control-planes list
- [x] apply

### Issues resolved

Fix #362

### CODE REVIEW

**Comment by yskopets** [comment]
@Gabitchov Hey!

Thanks a lot for getting started right away! 👍 

I've noticed that CI build is failing. It usually happens due failing code style checks, outdated auto-generated code, outdated auto-generated docs, etc.

To fix the build, you need to run `make check` before committing the changes. This command will run all necessary checks and generators.

**Comment by yskopets** [comment]
@Gabitchov By the way, the check that failed is actually useful in this context.

In order to overview how our `--help` commands look from user perspective (and catch any accidental changes to commands/options along the way), we record `--help` messages from all commands and store them in `git`.

The check failed because you changed the output of `kumactl` and didn't run `make check` to re-record all `--help` messages.

File: app/kumactl/cmd/install/install_control_plane.go [hunk 1]
```diff
@@ -62,9 +62,10 @@ func newInstallControlPlaneCmd(pctx *kumactl_cmd.RootContext) *cobra.Command {
 		SdsTlsKey:               "",
 	}
 	cmd := &cobra.Command{
-		Use:   "control-plane",
-		Short: "Install Kuma Control Plane on Kubernetes",
-		Long:  `Install Kuma Control Plane on Kubernetes.`,
+		Use:     "control-plane",
+		Short:   "Install Kuma Control Plane on Kubernetes",
+		Long:    `Install Kuma Control Plane on Kubernetes.`,
+		Example: `1. kumactl install control-plane | kubectl apply -f -`,
```
Comment (yskopets) [inline app/kumactl/cmd/install/install_control_plane.go]: Sorry, my example with `kumactl install database-schema` was misleading.

Let's use a slightly different format for `Example` field:
* Let's add a line with a short description for each example command
* Let's provide 2-3 examples for each command, e.g.
  * 1st example is the shortest or the most practical option
  * 2nd example to demonstrate how to override default values

E.g.,

```
	cmd := &cobra.Command{
		Use:   "control-plane",
		Short: "Install Kuma Control Plane on Kubernetes",
		Long:  `Install Kuma Control Plane on Kubernetes.`,

		Example: `
  # Install Kuma on Kubernetes
  kumactl install control-plane | kubectl apply -f -

  # Generate installation script for a target platform. At the moment, always generate Kubernetes YAML
  kumactl install control-plane`,

  		RunE: func(cmd *cobra.Command, _ []string) error {
                     ...
                },
	}
```
which will produce the following output:

```
Install Kuma Control Plane on Kubernetes.

Usage:
  kumactl install control-plane [flags]

Examples:

  # Install Kuma on Kubernetes
  kumactl install control-plane | kubectl apply -f -

  # Generate installation script for a target platform. At the moment, always generate Kubernetes YAML
  kumactl install control-plane

Flags:

  ...
```

**Comment by jakubdyszkiewicz** [comment]
Closing because of no activity. Due to the age of the PR, I think it has to be rewritten anyway.
Feel free to reopen the PR with current HEAD of master branch.

Thank you.

**Pull request closed without merging**